apps/src/templates/teacherNavigation/TeacherNavigationSidebarOption.tsx

```diff
@@ -0,0 +1,30 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import FontAwesomeV6Icon from '@cdo/apps/componentLibrary/fontAwesomeV6Icon/FontAwesomeV6Icon';
+import {BodyTwoText} from '@cdo/apps/componentLibrary/typography';
+
+import styles from './teacher-navigation.module.scss';
+
+interface TeacherNavigationSidebarOptionProps {
+  icon: string;
+  optionTitle: string;
+  isSelected: boolean;
+}
+
+const TeacherNavigationSidebarOption: React.FC<
```

Comment:
This is in the teacherNavigation folder and only used by the TeacherNavigationBar, what do you think about naming it just `SidebarOption` instead?